arangod/Aql/Functions.h

```diff
@@ -471,6 +471,11 @@ struct Functions {
   static AqlValue CurrentUser(arangodb::aql::ExpressionContext*,
                               transaction::Methods*, VPackFunctionParameters const&);
 
+  static AqlValue GetValidation(arangodb::aql::ExpressionContext*,
+                              transaction::Methods*, VPackFunctionParameters const&);
+  static AqlValue Validate(arangodb::aql::ExpressionContext*,
+                              transaction::Methods*, VPackFunctionParameters const&);
```

Comment:
These should be named like the AQL functions, i.e. `GetSchema` and `SchemaValidate`.